src/Types/IDEntitySts.php

```diff
@@ -25,17 +33,17 @@ public function getType(): string
     public static function normalize($value): ?string
     {
         try {
-            // Переводим в верхний регистр + trim
-            $value = Str::upper(\trim((string) $value));
+            // Uppercase + trim
+            $value = \mb_strtoupper(\trim((string) $value), 'UTF-8');
 
-            // Удаляем все символы, кроме разрешенных
+            // Remove all chars except allowed
             $value = (string) \preg_replace('~[^' . 'АБВГДЕЖЗИКЛМНОПРСТУФХЦЧШЩЫЭЮЯ' . 'A-Z' . '0-9]~u', '', $value);
 
-            // Производим замену латинских аналогов на кириллические (обратная транслитерация)
+            // Replace latin- chars with kyr- analogs (backward transliteration)
```

Comment:
Not important. Maybe change all `kyr` strings in comment and names of vars and constants to `cyrillic`? Diffucult for understand what `kyr` means.

Reply:
Yeah, my bad